google/auth/transport/requests.py

```diff
@@ -202,6 +199,7 @@ class _MutualTlsAdapter(requests.adapters.HTTPAdapter):
     def __init__(self, cert, key):
         import certifi
         from OpenSSL import crypto
+        from requests.packages.urllib3.util.ssl_ import create_urllib3_context
```

Comment:
Is it safe to assume that this import is available at this point?

Reply:
This class needs the `create_urllib3_context` function from the package, so if the import is not available we have to throw exceptions anyway.

`requests.packages.urllib3.util.ssl_` comes from the `requests` package, I don't know why it could be unavailable... This is more likely a `requests` package issue or user's python installation issue to me. Are you able to reproduce that problem?